eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageTypes.scala

```diff
@@ -66,7 +67,8 @@ case class AcceptChannel(temporaryChannelId: BinaryData,
                          revocationBasepoint: Point,
                          paymentBasepoint: Point,
                          delayedPaymentBasepoint: Point,
-                         firstPerCommitmentPoint: Point) extends ChannelMessage with HasTemporaryChannelId
+                         firstPerCommitmentPoint: Point,
+                         htlcBasepoint: Point) extends ChannelMessage with HasTemporaryChannelId
```

Comment:
field order has been changed, in the specs it is now:
    * [`33`:`payment_basepoint`]
    * [`33`:`delayed_payment_basepoint`]
    * [`33`:`htlc_basepoint`]
    * [`33`:`first_per_commitment_point`]